app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -4,7 +4,7 @@
   <h2 class="wpcc__heading">2. <%= t('wpcc.contributions.title') %></h2>
   <div class="contributions__row">
     <p class="t-contributions_description">
-      <%= t('wpcc.contributions.description', eligible_salary: @your_contribution.eligible_salary) %>
+      <%= t('wpcc.contributions.description', eligible_salary: number_with_delimiter(@your_contribution.eligible_salary, :delimiter => ',')) %>
```

Comment:
this was bundled in with other commits to keep a test working. For sure I can revisit this now and perhaps use a similar format to what we have elsewhere :)